examples/settings/Kconfig

```diff
@@ -0,0 +1,61 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+config EXAMPLES_SETTINGS
+	tristate "Settings example"
+	default n
+	---help---
+		Enable the "settings" example. This uses the functions provided in
+		utils/settings to test the storage and retrieval of various
+		configurable parameters
+
+if EXAMPLES_SETTINGS
+
+config EXAMPLES_SETTINGS_PROGNAME
+	string "Program name"
+	default "settings"
+	---help---
+		This is the name of the program that will be used when the NSH ELF
+		program is installed.
+
+config EXAMPLES_SETTINGS_PRIORITY
+	int "settings task priority"
+	default 100
+
+config EXAMPLES_SETTINGS_STACKSIZE
+	int "settings stack size"
+	default DEFAULT_TASK_STACKSIZE
+
+choice
+	prompt "Select settings storage location"
+	default EXAMPLES_SETTINGS_USE_TMPFS
+
+config EXAMPLES_SETTINGS_USE_TMPFS
+	bool "Use TMPFS"
+	select FS_TMPFS
+	---help---
+		TMPFS will be enabled and used
+
+config EXAMPLES_SETTINGS_USE_OTHER
+	bool "Use existing/other storage media"
+
+endchoice # Select settings storage location
+
+if EXAMPLES_SETTINGS_USE_OTHER
+
+config EXAMPLES_SETTINGS_EXISTING_STORAGE
+	string "Path to existing storage media"
+	default "/mnt/sdcard0"
```

Comment:
```suggestion
	default "/mnt"

Reply:
Normally /dev/mmcsd0 is mount at /mnt, also the user his other storage (flash, etc) to /mnt

Reply:
ok! done.